.eslintrc.js

```diff
@@ -7,6 +7,8 @@ module.exports = {
   },
   plugins: ['@typescript-eslint/eslint-plugin'],
   extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/eslint-recommended',
     'plugin:@typescript-eslint/recommended',
```

Comment:
hmm... interesting... what's the point of using both?
plugin:@typescript-eslint/eslint-recommended
plugin:@typescript-eslint/recommended